packages/registry/profiles/samsung/samsung-gearvr.json

```diff
@@ -0,0 +1,28 @@
+{
+    "profileId": "samsung-gearvr",
+    "fallbackProfileIds": [ "generic-trigger-touchpad"],
+    "layouts" : {
+        "none" : {
```

Comment:
Same issue as above. This device will report a left or right handedness based on platform configuration.